plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,135 @@
+package stomp
+
+import (
+	ts "crypto/tls"
+	"net"
+
+	"github.com/go-stomp/stomp"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+//STOMP ...
+type STOMP struct {
+	Host      string `toml:"host"`
+	Username  string `toml:"username,omitempty"`
+	Password  string `toml:"password,omitempty"`
+	QueueName string `toml:"queueName"`
+	SSL       bool   `toml:"ssl"`
+	tls.ClientConfig
+	Conn      *ts.Conn
+	NetConn   net.Conn
+	Stomp     *stomp.Conn
```

Comment:
I think you do not need to export this fields.

Reply:
done